Yoast/yoast-components
# Change translation method for SnippetEditorFields

## Summary

This PR can be summarized in the following changelog entry:

* _not applicable_

## Test instructions

You should view this change using the `&w=1` flag of GitHub. This means you can see the diff in the snapshots without whitespace changes. This makes it much easier to read.
